clientd3d/dialog.c

```diff
@@ -14,6 +14,8 @@
 #define PAGE_BREAK_CHAR '�'	  /* For multi-page descriptions */
 #define MAX_PAGE_DESCRIPTION_TEXT MAXMESSAGE
 
+static DescDialogStruct info;	  /* Info for current Description dialog */
```

Comment:
Did this need to be moved from the stack to a global variable?  This uses more memory and makes the lifetime of the variable longer than is necessary.